### PR TITLE
revert time zone to default

### PR DESCRIPTION
**Pull request by isaacmg410**
Revert time zone to default

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -13,9 +13,6 @@ class Application < Rails::Application
 
     config.active_job.queue_adapter = :delayed_job
 
-    # Default app time zone
-    config.time_zone = 'Madrid'
```
Comment (tramuntanal) [inline config/application.rb:17]: why not comment it and add a comment like:
"""
Can not force time_zone as all timestamps in DB are in UTC and ought to be updated if this config was applied.
"""

**Review by tramuntanal — CHANGES_REQUESTED** [review]

File: config/application.rb (hunk 1 above)
Comment (aitorlb) [reply config/application.rb:17]: @tramuntanal changes applied.

**Review by tramuntanal — COMMENTED** [review]
@isaacmg410 as explained here: https://makandracards.com/makandra/46009-working-with-or-without-time-zones-in-rails-applications I think that the best option is as stated in the chapter "When ActiveRecord's time zone logic is disabled, and a local time zone is configured":
```
config.time_zone = 'Madrid'
config.active_record.default_timezone = :local
config.active_record.time_zone_aware_attributes = false
```
We must check server's local time before applying this.

**Review by tramuntanal — COMMENTED** [review]
Let's accept this as a temporal solution and perform a new PR if required in the future. I'm annotating in our Basecamp.

**Comment by isaacmg410** [comment]
@tramuntanal So, can you accept the last changes made by @aitorlb ?

**Review by tramuntanal — APPROVED** [review]

**Comment by aitorlb** [comment]
@tramuntanal @isaacmg410 I'm not authorized to merge this pull request.

**Pull request merged**